sdk/storage/Azure.Storage.Blobs/api/Azure.Storage.Blobs.netstandard2.0.cs

```diff
@@ -56,6 +77,7 @@ public partial class BlobClientOptions : Azure.Core.ClientOptions
         public bool EnableTenantDiscovery { get { throw null; } set { } }
         public string EncryptionScope { get { throw null; } set { } }
         public System.Uri GeoRedundantSecondaryUri { get { throw null; } set { } }
+        public Azure.Storage.StorageAudience? StorageAudience { get { throw null; } set { } }
```

Comment:
The type should be `StorageAudience` or `BlobAudience`, but the property should just be `Audience` so the property name is the same across libraries.

Reply:
Renamed property `Audience`.